Formula/picat.rb

```diff
@@ -1,9 +1,9 @@
 class Picat < Formula
   desc "Simple, and yet powerful, logic-based multi-paradigm programming language"
   homepage "http://picat-lang.org/"
-  url "http://picat-lang.org/download/picat326_src.tar.gz"
-  version "3.2.6"
-  sha256 "7954396866e3e12944c09a7ca4123f9ce5d6193caecb8899b79cfd098ada399d"
+  url "http://picat-lang.org/download/picat328_src.tar.gz"
+  version "3.2#8"
```

Comment:
Weird version number, but it's what upstream is using: http://picat-lang.org/updates.txt

We did ship `3.2#6` as `3.2.6`, though. Maybe we should do the same?

Reply:
We've shipped `#` in the past. Only `3.2.6` was different, and not for any particular reason: https://github.com/Homebrew/homebrew-core/pulls?q=is%3Apr+picat+is%3Aclosed